embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -350,7 +360,7 @@ private String normalizeTableNameCase(JdbcInputConnection con, String tableName)
     private List<Integer> findIncrementalColumnIndexes(JdbcSchema schema, List<String> incrementalColumns)
         throws SQLException
     {
-        ImmutableList.Builder<Integer> builder = ImmutableList.builder();
+        final ArrayList<Integer> builder = new ArrayList<>();
```

Comment:
declare by interface is better? `final List<String> builder = new ArrayList<>()`

Reply:
I don't think so. It's just a local variable.

We may achieve some benefits from `List` if it's a signature of visible methods (argument or return type). But for just a local variable, there are no benefits.

Reply:
I think `builder` is not appropriate as `ArrayList` variable name (`indices` or `list`?).

Reply:
Ah, in this PR, I focused to minimize the diff so that reviewers can focus on the essential difference. It has already been a big change, but it was impossible to split this PR into smaller portions.

(I also thought that it's still a kind of builder to build an "unmodifiable" list by `Collections.unmodifiableList` below, but I know it's a lame excuse...)

If you think it's not good, let's do a refactoring in another PR. What do you think?

Reply:
> let's do a refactoring in another PR. What do you think?

That's a good idea.